### ISSUE TITLE
Resource Links

### ISSUE THREAD

**Issue by appsplash99** [codesONLY/JavaScriptONLY#3]
What do you think about adding another directory - but instead of notes(which the repo is currently doing well), privode with links for learning stuff : links like - js.info, MDN docs, etc.....

**Comment by sohamsshah** [comment]
@appsplash99 Thanks for the interest. 
This seems a good addition I suppose.
Can remove the ReferenceBooks and we have something as:
.
├── Resources
│   ├── WebsiteReferences
│   └── ReferenceBooks
│   └── YoutubeReferences

We can keep these Resources Updated. Will need your help in maintaining this for sure :)
Any suggestions are welcomed!
Would you like to link a PR for this?

**Comment by appsplash99** [comment]
I would love to help.
And Yes I Will, link a PR by this weekend...😊

**Comment by sohamsshah** [comment]
Hi @appsplash99 Are you working on this?

**Comment by appsplash99** [comment]
Yes, but your master branch is ahead of mine, how to i make it even with mine?

**Comment by sohamsshah** [comment]
@appsplash99 No worries. [This](https://devdojo.com/devdojo/how-to-sync-a-master-branch-from-a-fork) will be helpful to sync your fork. Let me know if you face any roadblocks!

**Comment by appsplash99** [comment]
i deleted the old fork, and now making a fresh fork form which the pull request will be made.
